radixdlt/src/test/java/org/radix/network2/messaging/MessageCentralImplTest.java

```diff
@@ -80,26 +105,29 @@ public void testSetup() {
 
 		// Curse you singletons
 		Universe universe = mock(Universe.class);
-		when(universe.getMagic()).thenReturn(0);
+		doReturn(0).when(universe).getMagic();
```

Comment:
what is the difference between `when(universe.getMagic()).thenReturn(0)`  and `doReturn(0).when(universe).getMagic();` ?

Reply:
It really has to do with the complicated and static way that Mockito uses to set up mocking statically with spy() methods.  See here for more details: http://sangsoonam.github.io/2019/02/04/mockito-doreturn-vs-thenreturn.html